tensorflow_addons/layers/README.md

```diff
@@ -3,9 +3,9 @@
 ## Maintainers
 | Submodule  |  Maintainers  | Contact Info   |
 |:---------- |:----------- |:------------- |
-| maxout |  |  |
+| maxout | @who who who | fsx950223@gmail.com |
```

Comment:
Could you change this to your GitHub username @fsx950223

Reply:
changed